modules/hello-world/examples/hello_world_t.py

```diff
@@ -3,18 +3,18 @@
 [source,python]
 ----
 """
-#tag::intro[]
-from couchbase.cluster import Cluster
+# tag::intro[]
```

Comment:
```suggestion
#tag::intro[]
```

Not sure if this is intentional. Suggesting to remove to match the closing tag which has no space. Although if the preference is to have a space on all `# tag` that would be fine too. I've never tried with a space.